administrator/components/com_media/resources/scripts/components/browser/browser.vue

```diff
@@ -117,6 +118,14 @@ export default {
         width: this.$store.state.showInfoBar ? '75%' : '100%',
       };
     },
+    NoMedia() {
+      const directories = this.$store.getters.getSelectedDirectoryDirectories;
+      const files = this.$store.getters.getSelectedDirectoryFiles;
+      if (directories.length + files.length === 0) {
+        return true;
+      }
+      return false;
+    },
```

Comment:
```suggestion
    isEmpty() {
      const directories = this.$store.getters.getSelectedDirectoryDirectories;
      const files = this.$store.getters.getSelectedDirectoryFiles;
      return [...directories, ...files].length ? false : true;
    },
```

Also change line 14 to `<h4 v-if="isEmpty">`

Reply:
I have made the changes you have suggested thank you so much.